### PR TITLE
add hide invite code eye to join community input

### PR DESCRIPTION
**Pull request by Dinek007**

### CODE REVIEW

File: packages/desktop/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.tsx [hunk 1]
```diff
@@ -17,6 +17,8 @@ const JoinCommunity = () => {
   const joinCommunityModal = useModal(ModalName.joinCommunityModal)
   const createCommunityModal = useModal(ModalName.createCommunityModal)
 
+  const [showPassword, setShowPassword] = useState<boolean>(false)
```
Comment (siepra) [inline packages/desktop/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.tsx:20]: I don't really like naming it 'password' as we don't have any passwords here. It should be more like 'revealInputValue' or smthg

File: packages/desktop/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.tsx [hunk 2]
```diff
@@ -55,6 +61,9 @@ const JoinCommunity = () => {
       isConnectionReady={isConnected}
       isCloseDisabled={!currentCommunity}
       hasReceivedResponse={Boolean(currentIdentity && !currentIdentity.userCertificate)}
+      showPassword={showPassword}
+      handleClickShowPassword={handleClickShowPassword}
+      isPasswordField={true}
```
Comment (siepra) [inline packages/desktop/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.tsx:66]: Do we really need this prop? Can't we just distinguish basing on 'communityOwnership' ?

File: packages/desktop/src/renderer/components/widgets/settings/InviteToCommunity.test.tsx [hunk 3]
```diff
@@ -58,14 +60,42 @@ describe('InviteToCommunity', () => {
...
             </div>
             <div
-              class="MuiGrid-root MuiGrid-item"
+              class="MuiGrid-root makeStyles-linkContainer-6 MuiGrid-item"
             >
               <p
                 class="MuiTypography-root MuiTypography-body2"
                 data-testid="invitation-code"
               >
                 •••••••••••••••••••••••••
```
Comment (siepra) [inline packages/desktop/src/renderer/components/widgets/settings/InviteToCommunity.test.tsx:69]: I think additional test case for revealed password could be useful

File: packages/desktop/src/renderer/components/widgets/settings/InviteToCommunity.tsx [hunk 4]
```diff
@@ -28,15 +31,34 @@ const useStyles = makeStyles(theme => ({
...
+  eyeIcon: {
+    margin: '5px',
+    top: '5px'
   }
 }))
 
 interface InviteFriendProps {
   communityName: string
   invitationUrl: string
+  showPassword: boolean
```
Comment (siepra) [inline packages/desktop/src/renderer/components/widgets/settings/InviteToCommunity.tsx:52]: Same as above: we don't have any 'password'

**Review by siepra — COMMENTED** [review]
Missing storybooks! You can make them interactive by providing useState and mocking 'handleClickShowPassword'

File: packages/desktop/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.tsx (hunk 1 above)
Comment (siepra) [reply packages/desktop/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.tsx:20]: When you name anything, always ask yourself a question "Will someone know what's going on here in three months?". Keeping variable and function names simple and literal is sometimes crucial

**Review by holmesworcester — APPROVED** [review]

**Review by siepra — APPROVED** [review]

**Pull request merged**